torch/distributed/distributed_c10d.py

```diff
@@ -14,6 +14,7 @@
     GatherOptions,
     ReduceOptions,
     ReduceScatterOptions,
+    AllToAllOptions,
```

Comment:
This import list was following alphabet order

Reply:
Will fix this in next update.